atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueServiceTest.java

```diff
@@ -736,6 +738,80 @@ public void testDeleteMultipleVersions() {
     }
 
     @Test
+    public void testDeleteRangeReverse() {
+        if (reverseRangesSupported()) {
+            putTestDataForMultipleRows(); // set up rows row0,row1,row2
```

Comment:
You could remove the comments if you renamed the method to `putTestDataForRowsZeroOneAndTwo`